backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/concept.html.ftl

```diff
@@ -1,54 +1,69 @@
+<#import "templates/table.html.ftl" as table>
+
+<#function prepareTableConnectorItems connectorItems>
+  <#return connectorItems
+    ?sort_by("name")
+    ?map(x -> {
+      "${idHeader}": x.id,
+      "${labelHeader}": x.label,
+      "${requiredColumnsHeader}": x.requiredColumns?sort_by("name")?join(', ')
+    })
+  />
+</#function>
+
 <@layout.layout>
-	<@layout.kid k="ID" v=c.id/>
-	<@layout.kv k="Label" v=c.label/>
-	<@layout.kv k="Type" v=c.class.simpleName/>
-	<@layout.kv k="Structure Parent" v=c.structureParent/>
-	<@layout.kv k="Elements" v=c.countElements()?string.number/>
-	<@layout.kc k="Selects">
-			<ul>
-			<#list c.selects?sort_by("name") as select>
-				<li>
-					<@layout.kid k="ID" v=select.id/>
-					<@layout.kv k="Label" v=select.label/>
-					<@layout.kv k="Type" v=select.class.simpleName/>
-					<@layout.kv k="Description" v=select.description/>
-				</li>
-			</#list>
-			</ul>
-		</@layout.kc>
-	<@layout.kc k="Connectors">
-	<#list c.connectors as connector>
-		<@layout.kid k="ID" v=connector.id/>
-		<@layout.kv k="Label" v=connector.label/>
-		<@layout.kv k="Validity Dates" v=connector.validityDates?join(', ')/>
-		<@layout.kc k="Table">
-		<a href="/admin-ui/datasets/${c.dataset.id}/tables/${connector.table.id}">${connector.table.name}</a>
-        </@layout.kc>
-		<@layout.kc k="Filters">
-			<ul>
-			<#list connector.collectAllFilters()?sort_by("name") as filter>
-				<li>
-					<@layout.kid k="ID" v=filter.id/>
-					<@layout.kv k="Label" v=filter.label/>
-					<@layout.kv k="Type" v=filter.class.simpleName/>
-					<@layout.kv k="Columns" v=filter.requiredColumns?sort_by("name")?join(', ')/>
-				</li>
-			</#list>
-			</ul>
-		</@layout.kc>
-		<@layout.kc k="Selects">
-			<ul>
-			<#list connector.selects?sort_by("name") as select>
-				<li>
-					<@layout.kid k="ID" v=select.id/>
-					<@layout.kv k="Label" v=select.label/>
-					<@layout.kv k="Type" v=select.class.simpleName/>
-					<@layout.kv k="Columns" v=select.requiredColumns?sort_by("name")?join(', ')/>
-				</li>
-			</#list>
-			</ul>
-		</@layout.kc>
-	</#list>
-	</@layout.kc>
+  <@breadcrumbs.breadcrumbs
+    labels=["Datasets", c.dataset.label, "Concepts", c.label]
+    links=[
+      "/admin-ui/datasets",
+      "/admin-ui/datasets/${c.dataset.id}",
+      "/admin-ui/datasets/${c.dataset.id}#Concepts"
+    ]
+  />
+  <@infoCard.infoCard
+    class="d-inline-flex"
+    title="Concept ${c.label}"
+    labels=["ID", "Label", "Type", "Structure Parent", "Elements"]
+    values=[c.id, c.label, c.class.simpleName, c.structureParent!"", c.countElements()?string.number]
+  />
+
+  <@accordion.accordionGroup class="mt-3">
+    <@accordion.accordion summary="Selects" infoText="${c.selects?size} entries">
+      <@table.table
+        columns=["id", "label", "simpleName", "description"]
+        items=c.selects?sort_by("name")?map(x -> {"id": x.id, "label": x.label, "simpleName": x.class.simpleName, "description": x.description!""})
```

Comment:
Die Ids können in den tabellen sehr lang werden.
Es ist praktisch, wenn man hier nur den letzten Part angezeigt bekommt (name) aber ein 
![image](https://user-images.githubusercontent.com/12283268/226966731-3e49be8b-a8e2-47b6-9bc8-745a709d827b.png)-Symbol hat, um die komplette ID in die Zwischenablage zu schieben